milabench/cli.py

```diff
@@ -125,10 +142,9 @@ def get_base_defaults(base, arch="none", run_name="none"):
                 "extra": "${dirs.base}/extra/${group}",
                 "cache": "${dirs.base}/cache",
             },
-            "arch": arch,
             "group": "${name}",
             "install_group": "${group}",
-            "install_variant": "${arch}",
+            "install_variant": "${system.arch}",
```

Comment:
I think this is how I should refer to `system.arch` right?

Reply:
I don't think it matters too much.
It is fine like this.